pkg/config/config.go

```diff
@@ -85,6 +85,7 @@ func init() {
 	BindEnvAndSetDefault("skip_ssl_validation", false)
 	Datadog.SetDefault("hostname", "")
 	Datadog.SetDefault("tags", []string{})
+	Datadog.SetDefault("split_tags", map[string]string{})
```

Comment:
`BindEnvAndSetDefault` ? see https://github.com/DataDog/datadog-agent/pull/2089

Reply:
did that in the rebase as well ^